docs/User-Guide_Getting-Started.md

```diff
@@ -2,43 +2,45 @@
 
 Please, make sure you have:
 
-- a proper power supply according to the board manufacturer requirements (basic usage example: 5V/2A with DC Jack barrel OR thick USB cable)
+-  a proper power supply according to the board manufacturer requirements (basic usage example: 5V/2A with DC Jack barrel OR **thick** USB cable)
 - a reliable SD card (see below "How to prepare a SD card?")
 
 # What to download?
 
 For each board we usually provide:
 
 - one CLI Debian **and** one CLI Ubuntu based server image,
-- one desktop Ubuntu Xenial **or** Debian Stretch
+- one desktop Ubuntu Bionic **or** Debian Buster
+
+Other builds (like Debian Stretch/Bullseye Ubuntu Disco/Eoan) are experimental and are unsupported.
 
 Some boards have different options due to their hardware specialities - router or IoT boards.
 
-# Legacy or mainline?
+# Legacy or current?
 
 Both kernels, where exists, are stable and production ready, but you should use them for different purpuses since their basic support differs:
 
 **legacy**: video acceleration, NAND support, connecting displays
```

Comment:
Since we drop 3.4.y ... this doesn't stands anymore. Video acceleration can be brought up only on 5.4+, nand support is (rare anyway) experimental and disabled by default.